libs/execution_context/ebpf_core.c

```diff
@@ -1210,7 +1210,12 @@ ebpf_core_get_pinned_object(_In_ const cxplat_utf8_string_t* path, _Out_ ebpf_ha
 
 Done:
```

Comment:
I think failure from `ebpf_handle_create` should be printed in this function, because the caller cannot ignore this error.

```
   retval = ebpf_handle_create(handle, (ebpf_base_object_t*)object);
   if (retval != ERROR_SUCCESS) {
      EBPF_LOG_MESSAGE_ERROR(EBPF_TRACELOG_LEVEL_ERROR,  EBPF_TRACELOG_KEYWORD_CORE, 
   "ebpf_handle_create failed", retval);
   }

Done:
   // Upon error, callers of this function will log the error.
   EBPF_LOG_EXIT();
   return retval;
```

Reply:
But the below function is already printing error message in EBPF_RETURN_RESULT(return_value)

**ebpf_handle_create**(_Out_ ebpf_handle_t* handle, _Inout_ ebpf_base_object_t* object)
{
... 
    EBPF_RETURN_RESULT(return_value);
}

Reply:
Added. Thanks